app/services/scholars_archive/ordered_parser_service.rb

```diff
@@ -0,0 +1,18 @@
+module ScholarsArchive
+  class OrderedParserService
+    def self.parse(labels)
+      labels ||= []
+      parsed_labels = []
```

Comment:
It looks like this variable is not being used anywhere, so I think we can remove it.